contrib/intel/jenkins/run.py

```diff
@@ -0,0 +1,83 @@
+import tests
+import subprocess
+import sys
+import argparse
+import os
+
+sys.path.append(os.environ['CI_SITE_CONFIG'])
+import ci_site_config
+
+fab = os.environ['FABRIC']#args.fabric
+brname = os.environ['BRANCH_NAME']#args.branchname
+bno = os.environ['BUILD_NUMBER']#args.buildno
+
+#this script is executed from the /tmp/ 
+#this is required so that mpi tests find a valid location on the test node
+#to execute tests incase the jenkins workspace is not created
+#on that node. It is not intended that any build files be stored in this location. 
+
+#run fi_info test
+def fi_info_test(mode):
+    cmdpath = "{}/{}/{}/{}/bin/".format(ci_site_config.install_dir, \
+               brname, bno, mode)
```

Comment:
tip: you may be able to use the new Python "fstrings" to make the formatting more readable.